### DISCUSSION TITLE
Recovered cases worlwide

### DISCUSSION THREAD

**Discussion by SSeanin** [owid/covid-19-data#2822] [Q&A] [answered]
Hi

Does the dataset include the number of recovered cases anywhere?

**Comment by lucasrodes** [comment, marked as answer]
Hi @SSeanin, thanks for the interest in our data.

Our dataset does not include recovered patients. 

You can fins all our hospitalization-related indicators on [our page](https://ourworldindata.org/covid-hospitalizations).
